terraform/modules/data-share-service/rds.tf

```diff
@@ -0,0 +1,35 @@
+resource "random_string" "rds_username" {
+  length  = 16
+  special = false
+}
+
+resource "random_password" "rds_password" {
+  length  = 16
+  special = false
+}
+
+resource "aws_kms_key" "rds_key" {
+  description         = "Key for RDS encryption"
+  enable_key_rotation = true
+}
+
```

Comment:
I was having a think about this, should we keep all the KMS keys in the KMS file?